docs/modules/ROOT/pages/lm-eval-tutorial.adoc

```diff
@@ -245,6 +245,133 @@ Specify extra information for the lm-eval job's pod.
 |Mount a PVC as the local storage for models and datasets.
 |===
 
+== Output of LMEvalJob
```

Comment:
The output of the lm-evaluation-harness is stored in the `results` field of the LMEvalJob's status subresource. It would be good to mention where to find it. And the LMEvalJob's status subresource has several fields, including podName, state, reason, message, etc. Explaining those fields would be good for users too.

Reply:
Yes! Great point - added in most recent commit